scss/tech.scss

```diff
@@ -26,17 +26,23 @@ $text-small: 12px;
   text-align: center;
   width: 200px;
 
-   &::before {
-     border: 1px solid $green;
-     border-radius: 8px;
-     right: 10px;
-     bottom: 10px;
-     content: '';
-     height: 100%;
-     position: absolute;
-     width: 100%;
-     z-index: -1;
-   }
+  &::before {
+    border: 1px solid $green;
+    border-radius: 8px;
+    right: 10px;
+    bottom: 10px;
+    content: '';
+    height: 100%;
+    position: absolute;
+    transition: bottom 100ms ease-out;
```

Comment:
you should transition the `right` property too. you just separate them using commas. also use seconds instead of `ms` to better understand the timing. 100ms is way too fast for the human eye, try `0.5s` instead